docs/version/stable-release.md

```diff
@@ -5,9 +5,35 @@ nav_order: 1
 parent: Version
 ---
 
+## Stable Release - v0.2.0
+This is the second public release of the RAPIDS Accelerator for Apache Spark. 
+The list of supported operations is provided [here](../configs.md#supported-gpu-operators-and-fine-tuning)
+
+Hardware Requirements: 
+
+	GPU Architecture: NVIDIA Pascal™ or better (Tested on V100, T4 and A100 GPU)
+	
+Software Requirements:
+
+	OS: Ubuntu 16.04 & gcc 5.4 OR Ubuntu 18.04/CentOS 7 & gcc 7.3
+	
+	CUDA & Nvidia Drivers: 10.1.2 & v418.87+, 10.2 & v440.33+ or 11.0 & v450.36+
+	
+	Apache Spark 3.0, 3.0.1
+	
+	Apache Hadoop 2.10+ or 3.1.1+ (3.1.1 for nvidia-docker version 2)
+	
+	Python 3.x, Scala 2.12, Java 8 
+
+## Download
```

Comment:
do you want to add v0.2 here to match the Download 0.1 below?

Reply:
Updated